CHANGELOG.md

```diff
@@ -19,20 +16,11 @@ In Besu version 20.10, properly formatted requests that have valid parameters (c
 
 Users of Web3J should note that many calls will now return a result with the error field containing the message whereas before a call would throw an exception with the error message as the exception message.   
```

Comment:
Not sure who the best person is to talk to about this but will web3j users need a list of specific calls or a link to the GI that changed this to be able to assess any impact to them?

Reply:
Thanks - that adds the context.